plugin/pkg/scheduler/schedulercache/cache.go

```diff
@@ -106,6 +106,20 @@ func (cache *schedulerCache) List(selector labels.Selector) ([]*v1.Pod, error) {
 	return pods, nil
 }
 
+func (cache *schedulerCache) FilteredList(podFilter PodFilter, selector labels.Selector) ([]*v1.Pod, error) {
```

Comment:
Instead of duplicating the code please do:

```
func (cache *schedulerCache) List(selector labels.Selector) ([]*v1.Pod, error) {
  return cache.filteredList(selector, alwaysTrue)
}

func (cache *schedulerCache) FilteredList(podFilter PodFilter, selector labels.Selector) ([]*v1.Pod, error) {
  return cache.filteredList(selector, podFilter)
}

func (cache *schedulerCache) filteredList(podFilter PodFilter, selector labels.Selector) ([]*v1.Pod, error) {
  // as you FilterList is now
}
```